### ISSUE TITLE
Cancel confirmation email is not needed or should at least be altered

### ISSUE THREAD

**Issue by Popolechien** [kiwix/cardshop#258]
Currently if a user cancels an order they get the [recipient_order_failed](https://github.com/kiwix/cardshop/blob/master/scheduler/src/templates/recipient_order_failed.html) message, which is a bit misleading ("_someone will reach out_" / "_Accept our apologies_").

I honestly think that sending a confirmation email is a bit overkill and unneeded so the default option should be canceled = no email.

Worst case scenario, an alternative `recipient_order_canceled` message could be a bit simpler:

```
{% extends "email_order_base.html" %}
{% from "macros.html" import tagged_message %}

{% block message %}
{% call tagged_message("error") %}Your order has been canceled{% endcall %}
<p>Dear {{ order.recipient.name }},<br />
<p>Just a quick note to confirm that your recent order of the {{ order.sd_card.name }} configuration has been canceled.
<p>Have a nice day</p>
<p>The Kiwix Team</p>
{% endblock %}
```

**Comment by kelson42** [comment]
@Popolechien Sounds kind of a duplicate of #227?!

**Comment by Popolechien** [comment]
@kelson42 Yes and no. #227 refers to emails sent to admins and asks that they only be notified of failed orders (but not order launches or cancellation). 

This ticket refers to users and asks that cancellation emails should not be sent (but order confirmations and failure notifications still would).

I guess we can blend both scenarios into a single ticket, but I'm not sure it's appropriate.

**Comment by kelson42** [comment]
@Popolechien It is indeed slightly different. Probably better to keep two tickets.
